operation/fhir-operation-erase/README.md

```diff
@@ -0,0 +1,343 @@
+# Operation: fhir-operation-erase 
+
+This document outlines the design and acceptance criteria for the *fhir-operation-erase*. 
+
+**API Path** 
+- `/fhir-server/api/v4/[RESOURCE]/[ID]/$erase`
+- `/fhir-server/api/v4/[RESOURCE]/$erase`
+
+|PATH Name|Required|Value|
+|------|-----|-----|
+|`RESOURCE`|required|The Resource Type|
+|`ID`|optional|The Resource ID, which implies the parameter logicalId must be set in the parameters object|
+
+**Method**
+- POST - We do not support **GET**.
+
+**Parameters**
+
+|Parameter Name|Type|Value|Description|
+|------|-----|-----|-----|
+|`reason`|required|string|the textual reason to add to the Audit Event. A maximum of a 1000 characters.|
+|`patient`|optional|string|the patient id to add to the Audit Event. Required, if the resource is being erased is within the Patient Compartment|
+|`id`|optional|string|Required, if the logical ID when executing an erase at the Resource Type level, else it must not be included.|
+|`version`|optional|integer|If included, the version of the resource is erased.|
+
+These values are passed as a FHIR Parameters Resource.
+
+**HTTP Response Codes**
+
+- 200 OK - The resource or the resource-version is permanently deleted.
+- 400 BAD_REQUEST - The request is malformed.
+- 401 NOT_AUTHENTICATED - The users is not authenticated.
+- 403 NOT_AUTHORIZED - The user is not an administrator in an authorized user's group.
+- 404 NOT_FOUND - The resource is not found.
+- 500 INTERNAL SERVER ERROR - Failed to process the request.
+
+**Examples**
```

Comment:
Would be nice to have an example of a deletion at the Resource Type level, and also of a non-Patient deletion.

Reply:
Added